src/components/toolbar/ToolBar.vue

```diff
@@ -1,99 +1,119 @@
-      >
-        <font-awesome-icon :icon="minusIcon" />
-      </b-button>
-      <b-button
-        class="btn btn-sm btn-secondary"
-        @click="scale = initialScale"
-        :disabled="scale === initialScale"
-        data-test="zoom-reset"
-        v-b-tooltip.hover
-        :title="$t('Reset to initial scale')"
-      >
-        {{ $t('Reset') }}
-      </b-button>
-      <span class="btn btn-sm btn-secondary scale-value">{{ Math.round(scale*100) }}%</span>
-    </div>
+      <div class="btn-group btn-group-sm mr-2" role="group" aria-label="Zoom controls">
+        <b-button
+          class="btn btn-sm btn-secondary"
+          @click="scale += scaleStep"
+          data-test="zoom-in"
+          v-b-tooltip.hover
+          :title="$t('Zoom In')"
+        >
+          <font-awesome-icon :icon="plusIcon" />
+        </b-button>
+        <b-button
+          class="btn btn-sm btn-secondary"
+          @click="scale = Math.max(minimumScale, scale -= scaleStep)"
+          data-test="zoom-out"
+          v-b-tooltip.hover
+          :title="$t('Zoom Out')"
+        >
+          <font-awesome-icon :icon="minusIcon" />
+        </b-button>
+        <b-button
+          class="btn btn-sm btn-secondary"
+          @click="scale = initialScale"
+          :disabled="scale === initialScale"
+          data-test="zoom-reset"
+          v-b-tooltip.hover
+          :title="$t('Reset to initial scale')"
+        >
+          {{ $t('Reset') }}
+        </b-button>
+        <span class="btn btn-sm btn-secondary scale-value">{{ Math.round(scale*100) }}%</span>
+      </div>
 
-    <div class="btn-group btn-group-sm mr-2" role="group" aria-label="Additional controls">
-      <b-button
-        class="btn btn-sm btn-secondary ml-auto"
-        data-test="panels-btn"
-        @click="panelsCompressed = !panelsCompressed"
-        v-b-tooltip.hover
-        :title="panelsCompressed ? $t('Show Menus') : $t('Hide Menus')"
-      >
-        <font-awesome-icon :icon="panelsCompressed ? expandIcon : compressIcon" />
-      </b-button>
+      <div class="btn-group btn-group-sm mr-2" role="group" aria-label="Additional controls">
+        <b-button
+          class="btn btn-sm btn-secondary ml-auto"
+          data-test="panels-btn"
+          @click="panelsCompressed = !panelsCompressed"
+          v-b-tooltip.hover
+          :title="panelsCompressed ? $t('Show Menus') : $t('Hide Menus')"
+        >
+          <font-awesome-icon :icon="panelsCompressed ? expandIcon : compressIcon" />
+        </b-button>
+
+        <b-button
+          class="btn btn-sm btn-secondary mini-map-btn ml-auto"
+          data-test="mini-map-btn"
+          @click="miniMapOpen = !miniMapOpen"
+          v-b-tooltip.hover
+          :title="miniMapOpen ? $t('Hide Mini-Map') : $t('Show Mini-Map')"
+        >
+          <font-awesome-icon :icon="miniMapOpen ? minusIcon : mapIcon" />
+        </b-button>
+      </div>
 
       <b-button
+        v-if="breadcrumbData.length > 0"
```

Comment:
I wonder if this `v-if` is necessary. The save button doesn't inherently have anything to do with breadcrumbs. The fact that the breadcrumbs exist means this is being loaded from the core app, which is where the save button would actually have an effect. So in essence, this `v-if` is a flag that's saying "if loaded in processmaker core".

Since clicking this button only emits an event, I wonder if it's safe to remove the `v-if` and always render it.

Reply:
It would be just a visual thing. Clicking the save button in stand-alone does nothing. Removing would be safe.